src/handlers/message.ts

```diff
@@ -108,7 +108,11 @@ async function handleIncomingMessage(message: Message) {
 	}
 
 	// GPT (only <prompt>)
-	if (!config.prefixEnabled) {
+
+	const selfNotedMessage = message.fromMe && message.hasQuotedMsg === false && message.from === message.to;
+
+
+	if (!config.prefixEnabled || (config.prefixSkippedForMe && selfNotedMessage)) {
```

Comment:
This if does not check if the message contains `!dalle` or any other command and is executed before the other, this causes that if you try generating an image with PREFIX_SKIPPED_FOR_ME set to true it will try to send you a chatGPT answer instead of a dalle one.

This can easily be fixed by moving this logic into the end. 
Example:


```
// Clear conversation context (!clear)
{!clear logic}

// AiConfig (!config <args>)
{!config logic}

// GPT (!gpt <prompt>)
{!gpt logic}

// DALLE (!dalle <prompt>)
{!dalle logic}

// GPT Self Chat (only <prompt>)
const selfNotedMessage = message.fromMe && message.hasQuotedMsg === false && message.from === message.to;

if (!config.prefixEnabled || (config.prefixSkippedForMe && selfNotedMessage)) {
	await handleMessageGPT(message, messageString);
	return;
}
```

Reply:
yea you are right, let me adjust that

Reply:
fixed